src/main/webapp/app/shared/layouts/navbar/navbar.component.ts

```diff
@@ -126,4 +154,25 @@ export class NavbarComponent implements OnInit, OnDestroy {
             }
         }
     }
+
+    /**
+     * check if exam already started
+     */
+    examHasStarted(): boolean {
+        return this.exam?.startDate ? this.exam.startDate.isBefore(this.serverDateService.now()) : false;
+    }
+
+    /**
+     * check if exam is over
+     */
+    examIsOver(): boolean {
+        return this.exam?.endDate ? this.exam.endDate.isBefore(this.serverDateService.now()) : true;
+    }
+
+    /**
+     * check if exam mode is active
+     */
+    examModeActive(): boolean {
+        return this.examHasStarted() && !this.examIsOver();
```

Comment:
Can this be done in one check? (something like this?)
```suggestion
    /**
     * check if exam mode is active
     */
    get examModeActive(): boolean {
        if (this.exam && this.exam.startDate && this.exam.endDate) {
            return this.serverDateService.now().isBetween(this.exam.startDate, this.exam.endDate);
        }
        return false;
    }
```
(Also, this could maybe be a getter.)

Reply:
yeah sure, this seems really good, thanks :)

Reply:
the getter part didn´t work though